wear/src/main/java/io/homeassistant/companion/android/sensors/TheaterModeSensorManager.kt

```diff
@@ -0,0 +1,64 @@
+package io.homeassistant.companion.android.sensors
+
+import android.content.Context
+import android.provider.Settings
+import android.util.Log
+import io.homeassistant.companion.android.common.sensors.SensorManager
+import io.homeassistant.companion.android.common.R as commonR
+
+class TheaterModeSensorManager : SensorManager {
+    companion object {
+        private const val TAG = "TheaterSensor"
+
+        val theaterMode = SensorManager.BasicSensor(
+            "theater_mode",
+            "binary_sensor",
+            commonR.string.sensor_name_theater_mode,
+            commonR.string.sensor_description_theater_mode,
+            "mdi:movie-open",
+            entityCategory = SensorManager.ENTITY_CATEGORY_DIAGNOSTIC,
+            updateType = SensorManager.BasicSensor.UpdateType.INTENT

```

Comment:
[On Discord](https://discord.com/channels/330944238910963714/377667559303938050/1033053536503726203) you mentioned that this sensor isn't linked to DND after all. Is there another intent the app is using for this? If not the `updateType` should be changed (even if this is not visible in the app yet).

Reply:
Ah forgot to update this bit, this one can actually update with the interactive sensor but it should have normal update type since its not direct

Reply:
I will also update bedtime mode to be more correct as the intents are linked to a different sensor.

Reply:
That would be a good comment if we get decide to add sensor descriptions on the watch

Reply:
ok updated the description as well for both sensors :)